packages/components/radio/src/RadioGroup.tsx

```diff
@@ -0,0 +1,42 @@
+import { useCallback } from "react";
+import { RadioGroupProps } from "./radio.types";
+import { RadioGroupStyled } from "./radio.styles";
+import Radio from "./Radio";
+
+const RadioGroup = ({ 
+  options, 
+  name,
+  selectedValue,
+  onChange 
+}: RadioGroupProps) => {
+    
+  const defaultValue = options.length > 0 ? options[0].value : '';
```

Comment:
This validation isn't needed, the default behaviour of the radio should be empty, selectedValue prop should be an obligatory valid string, and you just need to compare that prop on checked and its done.